launch_checklist.module

```diff
@@ -117,5 +138,572 @@ function launch_checklist_checklistapi_checklist_items() {
         '#description' => t('Ensure Spam prevention measures are in place.'),
       ],
     ],
+
+    // Section 4 SEO.
+    '04_seo' => [
+      '#title' => t('SEO'),
+      '#description' => t('<p>SEO settings for the Drupal site.</p>'),
+
+      'analytics' => [
+        '#title' => t('Analytics'),
+        '#description' => t('Verify Google Analytics or similar is installed and correct access has been given'),
+        'handbook_page' => [
+          '#text' => t('Google Analytics module'),
+          '#url' => $google_analytics_route,
+        ],
+      ],
+      'xml_sitemap' => [
+        '#title' => t('XML Sitemap'),
+        '#description' => t('Check that the XML sitemap module is enabled and a sitemap has been created.'),
+        'handbook_page' => [
+          '#text' => t('XML Sitemap'),
+          '#url' => $xml_sitemap,
+        ],
+      ],
+      'title_meta_tags' => [
+        '#title' => t('Page Titles and Meta Tags'),
+        '#description' => t('Ensure that page titles and meta tags are set up.'),
+        // 'handbook_page' => [
```

Comment:
The remainder of the `handbook_page`'s are commented out as this is still WIP and links/routes will need to be found for those.